Resources/Private/CodeTemplates/Extbase/gitattributes.t

```diff
@@ -0,0 +1,9 @@
+/Documentation export-ignore
+/Tests export-ignore
+/.editorconfig export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.php_cs export-ignore
+/.scrutinizer.yml export-ignore
+/.travis.yml export-ignore
+/ExtensionBuilder.json
```

Comment:
I don't understand why you add ExtensionBuilder.json here?

Reply:
Ups, I forgot the export-ignore in this line

Reply:
The whole extension i only needed in development...
No need to install it in LIVE installations...
??

Reply:
This is not for the extension_builder extension but for the generated extension code. I guess you missed that point ;)